openedx/course-discovery
# chore: update CSV Loader to  update the default time of timestamp attrs

This PR updates the default time to `00:00:00` for `end_time` & `registration_close_time` .

<img width="617" alt="image" src="https://github.com/openedx/course-discovery/assets/78806673/5c9a861a-ebfe-403e-a5a0-133fd01c10a3">


Testing:
- Run `populate_executive_education_data_csv` mgmt command after setting `GETSMARTER_CLIENT_CREDENTIALS` on local.
- Verify `End Time`, `Start Time` and `Registration Close Time` of courses.
